docs/core/docker/building-net-docker-images.md

```diff
@@ -131,10 +131,10 @@ The Docker client can be installed in:
 
 ### Getting the sample application
 
-The easiest way to get the sample is by cloning the [samples repository](https://github.com/dotnet/dotnet-docker-samples) with git, using the following instructions: 
+The easiest way to get the sample is by cloning the [samples repository](https://github.com/dotnet/dotnet-docker) with git, using the following instructions: 
```

Comment:
Please change to ".NET Core Docker repository."

Reply:
@JRAlexander 👍 Done.